src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1555,6 +1566,54 @@ class DocBaseViewer extends BaseViewer {
         // For documents of only 1 page, default thumbnails as closed
         return toggledState && numPages > 1;
     }
+
+    // Annotation overrides
+    getInitialAnnotationMode() {
+        return this.options.enableAnnotationsDiscoverability ? AnnotationMode.REGION : AnnotationMode.NONE;
+    }
+
+    initAnnotations() {
```

Comment:
Would it be worthwhile to offload this to a separate helper "component" and that accepts the `annotator` and `options` objects?